source/_integrations/mycroft.markdown

```diff
@@ -22,11 +22,26 @@ To use Mycroft in your installation, add the following to your `configuration.ya
 # Example configuration.yaml entry
 mycroft:
   host: 0.0.0.0
+notify:
```

Comment:
Let's split this into a separate section on how to set up notifications, and also split the configuration block.

Reply:
Ok, I will rewrite it.